stock_picking_product_barcode_report/controllers/main.py

```diff
@@ -22,7 +22,8 @@ def report_barcode(
     ):
         if type == "gs1_128":
             Gs1_128 = barcode.get_barcode_class("gs1_128")
-            gs1_182 = Gs1_128(str(value), writer=SVGWitoutTextWriter())
+            writer = SVGWitoutTextWriter(module_max_height=height)
+            gs1_182 = Gs1_128(str(value), writer=writer)
```

Comment:
Looks like a typo
```suggestion
            gs1_128 = Gs1_128(str(value), writer=writer)
```
Change next line too

Reply:
Change done!